SessionPlayer.js

```diff
@@ -12,29 +12,47 @@ export default class SessionPlayer extends Component {
 
         this.session =
             [ // A Session
+                { // A Guide
+                    guideName: 'RepeatWords',
+                    instructions: [
+                        { // An Instruction
+                            duration: [2.5],
+                            instructionsText: 'this is NOTHING to me!'
```

Comment:
This is totally fine for now. I was thinking that for the future, if we have lots of these different "parameters" to customize instructions, we might want to bundle them up as `instructionsMetadata` (tentative name) and then pass them in all-in-one into the `DisplayedInstruction` component and let each component pull out what sub-attribute they need.

But this is premature optimizing at the moment so I'm just thinking out loud.

Reply:
Sounds reasonable. Yep, let's do it in the future.